### PR TITLE
Nxp pxp vglite v8 dev

### PR DESCRIPTION
**Pull request by sebastienfagard**
### Description of the feature or fix

This pull-request re-enables in LVGL v8 the acceleration of Blit and Fill operations for i.MX RT platforms using the PXP and VGLite API for GPU. The features and bugfixes of src/gpu/ are aligned with the content of LVGL 7.10.1 in the MCUXpresso SDK 2.10 to be released in June.

### Checkpoints
- [ ] Follow the [styling guide](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md)
- [ ] Update [CHANGELOG.md](https://github.com/lvgl/lvgl/blob/master/docs/CHANGELOG.md)
- [ ] Update the documentation

### CODE REVIEW

**Comment by embeddedt** [comment]
Something weird may have happened with your branch here... there are unrelated commits regarding an `src/widgets/lv_anim_img` widget, which does not appear to match the upstream one in `src/extra/widgets/animimg`.

**Comment by sebastienfagard** [comment]
@embeddedt the commits that add src/widgets/lv_anim_img are actually an evolution of the original src/extra/widgets/animimg, thus the conflicting symbols at build time.
Can I remove src/extra/widgets/animimg to replace it by src/widgets/lv_anim_img ?

**Comment by embeddedt** [comment]
I think it would be best to remove `src/widgets/lv_anim_img` from this PR, and add the relevant changes to `src/extra/widgets/animimg` in another PR.

**Comment by sebastienfagard** [comment]
@embeddedt Hi, I updated the PR:
- removed the conflicting lv_anim_img widget, this will be part of another PR
- squashed the commits for the PXP & VGLite acceleration
- formatted the commit as requested using the Angular rules
Regards,
Seb

File: src/gpu/lv_gpu_nxp_pxp.c [hunk 1]
```diff
@@ -349,6 +340,12 @@ void lv_gpu_nxp_pxp_disable_recolor(void)
  */
 static void lv_gpu_nxp_pxp_run(void)
 {
+	lv_disp_t * disp = _lv_refr_get_disp_refreshing();
```
Comment (embeddedt) [inline src/gpu/lv_gpu_nxp_pxp.c]:
```suggestion
    lv_disp_t * disp = _lv_refr_get_disp_refreshing();
```

**Comment by embeddedt** [comment]
The changes look fine to me; I don't have the hardware so I can't test this.

File: src/gpu/lv_gpu_nxp_pxp.c (hunk 1 above)
Comment (sebastienfagard) [reply src/gpu/lv_gpu_nxp_pxp.c]: indent fixed. Thanks.

**Comment by kisvegabor** [comment]
Looks good to me too. I've tested on iMX RT 595 and they worked well. 
I'll merge it if the CI runs successfully.

**Comment by kisvegabor** [comment]
Merging, thank you, Sebastien!

**Comment by sebastienfagard** [comment]
You are welcome.
Thanks for the review and merge.

**Pull request merged**